content/en/apps/reference/app-settings/transitions.md

```diff
@@ -676,58 +675,96 @@ If this configuration is not set then the message defaults to what is set in the
 
 ### create_user_for_contacts
 
+Users are automatically created for certain contacts.  Both creating a new user for a new contact and replacing an existing user with a new user are supported.
+
+#### Configuration
+
+Several configurations are required in `app_settings` to enable the `create_user_for_contacts` transition.
+
+[Login by SMS]({{< ref "apps/reference/api#login-by-sms" >}}) must be enabled by setting the `token_login` configuration.
+
+The [`app_url` property]({{< ref "apps/reference/app-settings#app_settingsjson" >}}) must be set to the URL of the application. This is used to generate the token login link for the new user.
+
+##### Example
+```json
+"app_url": "https://demo.app.medicmobile.org",
+"token_login": {
+  "enabled": true,
+  "translation_key": "sms.token.login.help"
+},
+"transitions": {
+  "create_user_for_contacts": true
+}
+```
+
+#### Create User
+
+When adding a new contact, the `create_user_for_contacts` transition can be triggered to create a new user associated with that contact. _Available since 4.2.x._
+
+##### Example scenario
+
+This functionality would allow a supervisor to add a new contact for a new CHW and have a user for the new CHW automatically be created with the data from the new contact.
+
+Once the new contact is synced with the server and has been processed by Sentinel, the new CHW will receive an SMS message (at the phone number specified in the contact) containing a token login link. This link will allow them to login as the newly created user.
+
+##### Form Configuration
+
+When the `create_user_for_contacts` transition is enabled, Sentinel will attempt to create a user for any _newly created_ `person` contacts with the `user_for_contact.create` field set to `'true'`. So, `contact` forms and `app` forms for adding persons that should trigger new user creation need to include a `user_for_contact` group that contains a `create` field. The calculation for the value of the `create` field should evaluate to `'true'` when a new user is desired.  Any other value for that field will not trigger user creation.
```

Comment:
Is this a truthy  `'true'` string or a boolean `true`?  I don't think it much matters, but I was reading it closely and figured I'd ask!

Reply:
Yeah, good question!  It is a `'true'` string because that is how Enketo (ODK?) serializes boolean values.  Not sure exactly why Enketo outputs string values instead of real booleans (which are supported in JSON), but this follows existing patterns we have in other places and keeps the form implementation as simple as possible, so seems okay.